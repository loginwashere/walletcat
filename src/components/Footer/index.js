import React, { Component } from 'react';
import {
  Nav,
  Navbar,
  NavItem
} from 'react-bootstrap';
import { LinkContainer } from 'react-router-bootstrap';

export class Footer extends Component {
  render() {
    return (
      <Navbar collapseOnSelect fixedBottom>
        <Navbar.Toggle />
        <Navbar.Collapse>
          <Nav>
            <LinkContainer to="/about">
              <NavItem eventKey={7} href="#">About</NavItem>
            </LinkContainer>
            <LinkContainer to="/contact-us">
              <NavItem eventKey={8} href="#">Contact Us</NavItem>
            </LinkContainer>
          </Nav>
        </Navbar.Collapse>
      </Navbar>
    )
  }
}

export default Footer;