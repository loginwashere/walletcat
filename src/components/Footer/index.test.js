import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '.'

describe('components:Footer:', () => {
  it('renders without crashing', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
