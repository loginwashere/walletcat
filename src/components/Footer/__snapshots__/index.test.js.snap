exports[`components:Footer: renders without crashing 1`] = `
<nav
  className="navbar navbar-default navbar-fixed-bottom">
  <div
    className="container">
    <button
      className="navbar-toggle collapsed"
      onClick={[Function]}
      type="button">
      <span
        className="sr-only">
        Toggle navigation
      </span>
      <span
        className="icon-bar" />
      <span
        className="icon-bar" />
      <span
        className="icon-bar" />
    </button>
    <div
      aria-expanded={null}
      className="navbar-collapse collapse">
      <ul
        className="nav navbar-nav"
        role={null}>
        <li
          className=""
          role="presentation"
          style={undefined}>
          <a
            action="push"
            href="#"
            onClick={[Function]}
            role="button">
            About
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}>
          <a
            action="push"
            href="#"
            onClick={[Function]}
            role="button">
            Contact Us
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}>
          <a
            action="push"
            href="#"
            onClick={[Function]}
            role="button">
            Privacy Policy
          </a>
        </li>
        <li
          className=""
          role="presentation"
          style={undefined}>
          <a
            action="push"
            href="#"
            onClick={[Function]}
            role="button">
            Terms and conditions
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
